Packs/DeepInstinct/Integrations/DeepInstinct/DeepInstinct.yml

```diff
@@ -0,0 +1,179 @@
+commonfields:
+  id: Deep Instinct
+  version: -1
+name: Deep Instinct
+display: Deep Instinct
+category: Network Security
+description: Deep Instinct
+configuration:
+- display: base server url
+  name: base_url
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: api key
+  name: apikey
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Fetch incidents
+  name: isFetch
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  type: 13
+  required: false
+- display: first event id to fetch from
```

Comment:
how do i get the first event id?

Reply:
What do you mean? default is the first one(id=0) other than that, you can see you ids from events commands(or fetch events)

Reply:
As discussed with Eldad, the convention in demisto is using date(Or how much back in time to look for), But I understood that your API does not support it, so you can dismiss this comment.